src/main/java/com/example/hop_oasis/hendler/ValidationHandler.java

```diff
@@ -31,7 +32,8 @@ public ResponseEntity<ErrorDetails> handleValidationExceptions(HttpServletReques
     }
 
     @ResponseStatus(HttpStatus.BAD_REQUEST)
-    @ExceptionHandler({BeerNotFoundException.class, ImageNotFoundException.class})
+    @ExceptionHandler({BeerNotFoundException.class, ImageNotFoundException.class, SnackNotFoundException.class,
+                      ProductBundleNotFoundException.class})
```

Comment:
You can just create more general EntityNotFoundException or ResourceNotFoundException, or DatabaseItemNotFoundException, or come up with other name, and use it instead of all of these exceptions